### PR TITLE
Added service serving certificate list

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1710693

Redo of https://github.com/openshift/openshift-docs/pull/20999, where I got merge happy :woman_facepalming: 

Preview Build: http://file.rdu.redhat.com/~ahardin/April-15-2020/service-serving-certs/dev_guide/secrets.html#service-serving-certificate-secrets

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@gpei PTAL Thank you!

**Comment by ahardin-rh** [comment]
@wsun1 PTAL. Thank you!

**Comment by wsun1** [comment]
@xingxingxia @yapei @anpingli @juzhao @emmajiafan  Please review this PR, thanks.

File: dev_guide/secrets.adoc [hunk 1]
```diff
@@ -225,6 +225,24 @@ applications that need out-of-the-box certificates. It has the same settings as
...
+openshift-infra	                  heapster-certs
+openshift-logging	                prometheus-tls
+openshift-monitoring	            alertmanager-main-tls
+openshift-monitoring	            grafana-tls
+openshift-monitoring            	kube-state-metrics-tls
+openshift-monitoring            	node-exporter-tls
+openshift-monitoring	            prometheus-k8s-tls
+openshift-template-service-broker	apiserver-serving-cert
+openshift-web-console	            webconsole-serving-cert
+----
```
Comment (juzhao) [inline dev_guide/secrets.adoc:244]: no issue for openshift-monitoring secrets
FYI: It is better to have a good format for all the secrets

**Comment by xingxingxia** [comment]
AFAIK, the service serving area has two aspects: the service serving certificates CA, and the service serving certificates themselves. The latter exist in subteams' components. For the former, @barleyer , please help comment if you think this PR has concerns about it. Thanks

**Comment by barleyer** [comment]
> AFAIK, the service serving area has two aspects: the service serving certificates CA, and the service serving certificates themselves. The latter exist in subteams' components. For the former, @barleyer , please help comment if you think this PR has concerns about it. Thanks

THis is for 3.11.

File: dev_guide/secrets.adoc (hunk 1 above)
Comment (ahardin-rh) [reply dev_guide/secrets.adoc]: @juzhao Thank you! I reformatted the information as a table:
http://file.rdu.redhat.com/~ahardin/April-15-2020/service-serving-certs/dev_guide/secrets.html#service-serving-certificate-secrets

**Comment by ahardin-rh** [comment]
@yapei @anpingli @emmajiafan PTAL. Thank you!

**Comment by bobfuru** [comment]
LGTM!

**Comment by yapei** [comment]
LGTM

**Comment by emmajiafan** [comment]
LGTM

**Comment by anpingli** [comment]
LGTM  Logging

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**